feeds/models.py

```diff
@@ -34,6 +34,24 @@ def link(self):
             if self.url and link:
                 link = parse.urljoin(self.url, link)
             return link
+            
+    @property
+    def categories(self):
+        return self.tree.findall('{}category'.format(self.defaultns))
```

Comment:
We can probably just have this return the text from each of the elements, instead of returning the element. That's how the ones above operate as well.